### PR TITLE
Add delete codec

### PR DESCRIPTION
**Pull request by godchen0212**
Add delete codec
Issue #7130 

Signed-off-by: godchen <qingxiang.chen@zilliz.com>

### CODE REVIEW

File: internal/storage/data_codec.go [hunk 1]
```diff
@@ -134,6 +135,12 @@ type InsertData struct {
 	Infos []BlobInfo
 }
 
+// DeleteData saves each entity delete message represented as <pk,ts> map.
+// ts represents the time when this instance was deleted
+type DeleteData struct {
+	Data map[string]int64 // field id to field data
```
Comment (sunby) [inline internal/storage/data_codec.go:141]: field id ?

**Comment by yanliang567** [comment]
manually label ci-passed,  will add more ut after some refactor work

**Pull request merged**